CONTRIBUTING.md

```diff
@@ -142,9 +148,55 @@ If you plan to fetch GitHub OWASP data locally, follow these additional steps:
 1. **Sync Local Database Data**:
    - Now you should be able to run the following command to sync your local database data with GitHub:
 
-   ```bash
-   make sync
-   ```
+      ```bash
+      make sync
+      ```
+
+#### NestBot Development
+
+To setup NestBot development environment, follow these steps:
+
+1. **Set Up ngrok**:
+   - Go to [ngrok](https://ngrok.com/) and create a free account.
+   - Install and configure ngrok on your machine using these [instructions](https://ngrok.com/docs/getting-started/#step-1-install)
+   - Create your static domain by simply going to [ngrok domains](https://dashboard.ngrok.com/domains)
+   - Run the following commands to edit ngrok configuration:
+
+     ```bash
+     ngrok config edit
+     ```
+
+     ```plaintext
+     tunnels:
+        NestBot:
+          addr: 8000
+          proto: http
+          hostname: <your-static-domain>
+     ```
+
+   - Now ngrok is all set, you access your local setup over internet, running the follwing command:
+
+     ```bash
+     ngrok start NestBot
```

Comment:
@ShashaankS have you tested these set up instructions?

Reply:
Yes, I have tested it

![Screenshot from 2024-12-31 00-31-04](https://github.com/user-attachments/assets/bc2c4a2f-e8e9-4e66-ba07-d3bbb63b760d)

This is my config file and it works well.

Reply:
It's a bad idea to share your auth tokens.

What ngrok plan do you have?

Reply:
I got you, I won't share them.
Its the free one.

Reply:
Okay, I'll check my config later and let you know.